lib/di/interceptor/app_interceptor.dart

```diff
@@ -2,25 +2,26 @@
 import 'dart:io';
 
 import 'package:dio/dio.dart';
+import 'package:flutter_survey/utils/storage/secure_storage.dart';
 
 const String _authorizationHeader = 'Authorization';
 
 class AppInterceptor extends Interceptor {
   final bool _requireAuthentication;
   final Dio _dio;
+  final SecureStorage _secureStorage;
 
-  AppInterceptor(
-    this._requireAuthentication,
-    this._dio,
-  );
+  AppInterceptor(this._requireAuthentication, this._dio, this._secureStorage);
 
   @override
   Future onRequest(
       RequestOptions options, RequestInterceptorHandler handler) async {
     if (_requireAuthentication) {
-      // TODO: Integrate log-in https://github.com/nimblehq/ic-flutter-taher-toby/issues/10
-      options.headers.putIfAbsent(
-          _authorizationHeader, () => "Bearer add your token here");
+      final String bearerToken =
+          await _secureStorage.readSecureData(accessTokenKey) ??
+              'Add your token here';
+      options.headers
+          .putIfAbsent(_authorizationHeader, () => 'Bearer $bearerToken');
```

Comment:
There is a `token_type` field from the response, we should save and load it instead of hard-coding the `Bearer` value

![image](https://user-images.githubusercontent.com/16315358/232662779-2d7957ee-2f50-45e0-82ba-7137dae972a2.png)